mobify/mobify-code-style
# Add a stylelint ruleset

Status: **WIP**
Owner: @marlowpayne
Reviewers: @jeffkamo @mikenikles @jansepar 

## Changes
- SCSS-lint is the very last tool in our build chain that requires Ruby
- This change is in tandem with this generator PR: https://github.com/mobify/generator-adaptivejs/pull/343 , which migrates SCSS linting to [stylelint](http://stylelint.io/), finally breaking our dependency on Ruby
- This ruleset needs to be brought up to the feature parity with our existing [SCSS linting ruleset](https://github.com/mobify/mobify-code-style/blob/develop/css/.scss-lint.yml) to effectively enforce our SCSS code style

## Todos:
- [ ] Reach feature parity (within acceptable bounds) with our existing SCSS linting ruleset
- [ ] UI Dev +1

### Feedback:
_none so far_

## How to Test
- Check out this generator PR, which uses this ruleset: https://github.com/mobify/generator-adaptivejs/pull/343
